### ISSUE TITLE
NTR, pulmo: Organizing Hyaline Membranes

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6397]
**Preferred term label:**
Organizing Hyaline Membranes

**Synonyms**
Presence of hyaline membranes

**Definition (free text, please give PubMed ID)**
hyaline membranes are made up of dead cells, surfactant, and proteins. The hyaline membranes deposit along the walls of the alveoli, where gas exchange typically occurs, thereby making gas exchange difficult.

ISBN 978-1-4511-7643-8
Churg, Andrew, and Churg. Atlas of Interstitial Lung Disease Pathology : Pathology with High Resolution CT Correlations, Wolters Kluwer, 2013. 

**Parent term (use hpo.jax.org/app)**
Interstitial pulmonary abnormality HP:0006530

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:79126   (Disorder)   Acute interstitial pneumonia
ORPHA:1302   (Disorder)   Cryptogenic organizing pneumonia

**Your nano-attribution (ORCID)**
LMU:chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
@GrieseAG  -- we need to choose a term label that is comprehensible in isolation "Organizing Hyaline Membranes" is not necessarily lung

**Comment by pnrobinson** [comment]
Also, is this interstitial or alveolar?

**Comment by pnrobinson** [comment]
Parent: Abnormal pulmonary alveolar system morphology HP:0033137
Label:Organizing hyaline membranes

synonym: Organization of hyaline membranes

**Comment by pnrobinson** [comment]
@GrieseAG  I am wondering about "organizing" -- Is there a case to be made for a term "Hyaline membranes" (without "organizing").
Slightly revised definition -- 
Hyaline membranes are homogeneous eosinophilic material composed of cellular debris, plasma proteins, and surfactant
plastered against alveolar ducts and alveolar walls. The hyaline membranes are deposited along the walls of the alveoli, where gas exchange typically occurs, thereby making gas exchange difficult. PMID:24224590


The organization phenotype appears to be separate -- 
 In the organizing stage, fibroblasts start to permeate through the alveolar duct walls to replace the hyaline membrane completely and to form membranous fibrous tissue. (https://pubmed.ncbi.nlm.nih.gov/7679849/).

**Comment by pnrobinson** [comment]
added term. Let's make new issues for "organization" terms as needed.
